noxfile.py

```diff
@@ -51,7 +51,19 @@ def tests_impl(session, extras="socks,secure,brotli"):
 
 
 @nox.session(
-    python=["2.7", "3.6", "3.7", "3.8", "3.9", "3.10", "3.11", "3.12", "3.13", "pypy"]
+    python=[
+        "2.7",
+        "3.6",
+        "3.7",
+        "3.8",
```

Comment:
Should we remove support for those in future work?

Reply:
I'm aware of #3410 and don't have a strong opinion on this, but I would prefer not to drop support for older Python versions in 1.26.x unless we encounter significant CI hurdles

Reply:
To be clear I specifically meant 3.6, 3.7 and maybe 3.8, not 2.7. Regarding #3410, I'd like to actually drop Python 2 entirely at some point, not only CI.